### PR TITLE
more words to the abstract and intro section

### LINKED ISSUE #2: Abstract & introduction
The Abstract/Introduction sections are fairly thin - we should add more details about the expected use cases.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #2

Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

File: draft-ftbs-rats-msg-wrap.md [hunk 1]
```diff
@@ -45,21 +45,57 @@ normative:
...
+
+In order to minimize the cost associated with registration and maximize
+interoperability, it is desirable to reuse their typing information
+across such boundaries.
+
 This document defines two encapsulation formats for RATS conceptual
-messages ({{Section 8 of -rats-arch}}).
+messages that aim to achieve the goals stated above.
+
+These encapsulation formats are designed to be:
```
Comment (nedmsmith) [inline draft-ftbs-rats-msg-wrap.md:83]: Wording should say "there are two formats" rather than "designed to have two formats"

**Review by nedmsmith — COMMENTED** [review]
minor wording issue

File: draft-ftbs-rats-msg-wrap.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:83]: I don't understand the editorial suggestion.  It doesn't seem to apply to L83.

**Review by hannestschofenig — APPROVED** [review]

File: draft-ftbs-rats-msg-wrap.md (hunk 1 above)
Comment (ionut-arm) [reply draft-ftbs-rats-msg-wrap.md:83]: Sounds like the suggested change would be along the lines of
```suggestion
There are two encapsulation formats available:
```

Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:83]: yes, but that doesn't work with what follows: the bulleted list describes the design criteria of the encaps, not the encaps themselves.

Comment (ionut-arm) [reply draft-ftbs-rats-msg-wrap.md:83]: Ah, right, I see now!

Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:83]: closing this and merging.

**Pull request merged**